atat_provisioning_wizard_api.yaml

```diff
@@ -1191,9 +1191,17 @@ components:
                 pop_end_date: "2023-09-01"
     ApplicationStepEx:
       value:
+        operators:
+          - display_name: "Yoda"
+            email: "yoda@iam.mil"
+            access: 'portfolio_administrator'
```

Comment:
```suggestion
            access: "portfolio_administrator"
```

Reply:
Thanks, I was just updating it, but you beat me to it. I have a habit of using single quotes.